components/script/dom/htmltableelement.rs

```diff
@@ -398,16 +398,20 @@ impl HTMLTableElementMethods for HTMLTableElement {
     // https://html.spec.whatwg.org/multipage/#dom-table-deleterow
```

Comment:
I think it would be good to fix the rustdoc in this function when we update it:

```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-table-deleterow>
```